plugin/src/main/kotlin/io/ktor/plugin/features/Ktor.kt

```diff
@@ -36,11 +40,38 @@ inline fun <reified T : Task> TaskContainer.registerKtorTask(
     }
 }
 
-abstract class KtorExtension
+abstract class KtorExtension @Inject internal constructor(
+    objects: ObjectFactory,
+    providers: ProviderFactory,
+) {
+
+    /**
+     * Enables a special mode targeted for development.
```

Comment:
Why do we need this description here? This description may become outdated if something changes in what the development mode affects.

Reply:
Okay, I'll remove the description of capabilities and keep only the link to the documentation